src/zenml/enums.py

```diff
@@ -144,6 +144,7 @@ class CliCategories(StrEnum):
     HUB = "ZenML Hub"
     INTEGRATIONS = "Integrations"
     MANAGEMENT_TOOLS = "Management Tools"
+    MODEL_WATCHTOWER = "Model Watch Tower"
```

Comment:
```suggestion
    MODEL_WATCHTOWER = "Model Watchtower"
```

That said, we should probably agree / align on the canonical way for us to refer to this so it's standardised within ZenML...